<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Scroll">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>431</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="TopicTextView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="TopicShowInNewWindow">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed{
	background-color: #00FF00;
}</string>
        </property>
        <property name="text">
         <string>Show in new window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TopicEdit">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed{
	background-color: #00FF00;
}</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="TopicHistory">
        <property name="styleSheet">
         <string notr="true">QPushButton:pressed{
	background-color: #00FF00;
}</string>
        </property>
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="QTreeView" name="TreeView">
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed</set>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionConnect_server"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Topic"/>
   <addaction name="separator"/>
   <addaction name="actionDashboard"/>
   <addaction name="separator"/>
   <addaction name="actionSimulate"/>
  </widget>
  <action name="actionConnect_server">
   <property name="text">
    <string>Connect server</string>
   </property>
  </action>
  <action name="actionNew_Topic">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Topic</string>
   </property>
  </action>
  <action name="actionDashboard">
   <property name="text">
    <string>Dashboard</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="text">
    <string>Start simulation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
